src/components/ha-config-entry-picker.ts

```diff
@@ -59,6 +59,7 @@ class HaConfigEntryPicker extends LitElement {
     >
     <span slot="secondary">${item.localized_domain_name}</span>
     <img
+      alt=""
```

Comment:
The localized domain name is rendered just prior to this image, therefore I believe the image to be decorative.